server/notification-providers/onechat.js

```diff
@@ -0,0 +1,88 @@
+const NotificationProvider = require("./notification-provider");
+const axios = require("axios");
+const { DOWN, UP } = require("../../src/util");
+
+class OneChat extends NotificationProvider {
+    name = "OneChat";
+
+    /**
+     * @inheritdoc
+     */
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+        const okMsg = "Sent Successfully.";
+        const url = "https://chat-api.one.th/message/api/v1/push_message";
+
+        try {
+            const config = {
+                headers: {
+                    "Content-Type": "application/json",
+                    Authorization: "Bearer " + notification.accessToken,
+                },
+            };
+            // Send a test message if the monitor is null
```

Comment:
Lets only have comments which are significant ^^

There are a few other cases of this

```suggestion
```